### PR TITLE
Update the invoice page message

### PR DESCRIPTION
**Pull request by kunalbotla**
The current message isn't clear about what options sponsors have for paying invoices.

image

### CODE REVIEW

File: app/views/invoices/new.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
   </p>
 <% else %>
   <p class="h5 line-height-3 muted mt0 mb2">
-    If the sponsor wants to pay via a mailed check or direct deposit instead of
+    Invoices can be paid with a card or bank transfer (ACH/wire). If the sponsor wants to pay via a mailed check instead of
     the online invoice:
```
Comment (garyhtou) [inline app/views/invoices/new.html.erb]: thoughts on this copy edit?

```suggestion
    Invoices can be paid with a card or bank transfer (ACH / Wire). If your
    sponsor wants to pay via a mailed check instead, please reach out to the
    Hack Club Bank team and we'll help you coordinate the receival of the check.
```

Comment (kunalbotla) [reply app/views/invoices/new.html.erb]: I think this looks good.

**Review by garyhtou — APPROVED** [review]
merge when ready!

**Comment by kunalbotla** [comment]
i thought i already merged this 😳

**Pull request merged**